dozer-cli/src/simple/orchestrator.rs

```diff
@@ -66,6 +67,10 @@ impl SimpleOrchestrator {
 
 impl Orchestrator for SimpleOrchestrator {
     fn run_api(&mut self, shutdown: ShutdownReceiver) -> Result<(), OrchestrationError> {
+        describe_histogram!(
+            dozer_api::API_LATENCY_HISTOGRAM_NAME,
+            "The api processing latency in miliseconds"
```

Comment:
Let's change it to in seconds to be consistent with other dozer latency metrics, as stated above